torch/onnx/symbolic.py

```diff
@@ -216,8 +216,6 @@ def mean(g, self, dim=None, keepdim=None):
 
 
 def sum(g, self, dim=None, keepdim=None):
-    if dim is None and keepdim is None:
```

Comment:
aren't you removing the dim=None case, which translates to Sum?

Reply:
does `g.op("ReduceSum"` handle case when axes_i=None and does a Sum over all dimensions (and return a single number)?

Reply:
It should never translate to Sum. If dim is missing, then ONNX ReduceSum sums over all, which matches what PyTorch does.

Reply:
Good point, updated to apply the fix to prod and mean